api/services/PassportService.js

```diff
@@ -203,11 +203,12 @@ module.exports = class PassportService extends Service {
 
     criteria[fieldName] = identifier
 
-    return this.app.services.FootprintService.find('User', criteria, {populate: 'passports', findOne: true})
+    return this.app.services.FootprintService.find('User', criteria, {populate: 'passports'})
```

Comment:
No, I was curious :) it's fine for me this way. Let's wait TravisCI and merge this ! :D thanks a lot for this !